### DISCUSSION TITLE
Bug: I can only download 13 items

### DISCUSSION THREAD

**Discussion by wirescarryingme** [TumblThreeApp/TumblThree#271] [General]
**Describe the bug**
I am trying to download a tumblr blog. The amount of items on the blog is 58 according to the application and It downloaded 13 items before freezing on 13. 

**To Reproduce**
1. start downloading a blog 
2. take note of the number of downloadable items and number of items downloaded
3. after an hour or so, The number will have been on 13 items downloaded for an hour

**Expected behavior**
I expected all items on the blog to download instead of only some of the blog

**Desktop (please complete the following information):**
 - TumblThree version: Latest version in releases as of 2022-08-16 (2022 august 16th)
 - OS: Windows 10
 - Browser Firefox

**Comment by thomas694** [comment]
Please give us some more details.
Does the download process freeze, the blog stays in the queue list and remains active/green in the blog list, or does it just not download the expected number of items but complete the download?
In the second case, did you check the numbers of the different item types in the details pane? Have you changed settings so that only some types are downloaded?
You can have a look in the log file (%LOCALAPPDATA%\TumblThree\TumblThree.log), whether there's an error.

**Reply by Linktydraa** [reply]
I have a similar issue. The application actually does both. It'll stay green and "download" a blog for a very long time, but it the number of items downloaded never seems to change, and then it'll suddenly be "complete" and then go off the queue.

the second issue that you mention happens too. I'll have a blog that says 9000+ downloadable items, and under the detail, I checked "Download images", "Download Video", and "download reblogged posts" but it only downloads 28 items. I never get an error message. It just doesn't download all the items it should. I know for a fact that there are more then 28 images on the particular blog I want to download from.


Here is one of the blogs that doesn't download fully. Warning: Probably NSWF

https://rawworld.tumblr.com/

**Reply by thomas694** [reply]
Can you give us more information?
After around how many images it stops? Or what is the post date (or oldest file date) when it stops?
Can you find any errors in the log file?

**Reply by Linktydraa** [reply]
It depends. For this blog in particular it downloaded 175 out of 72302. I understand that there might be some non-photo posts that it's skipping, but I went to the blog and just scrolled and scrolled for a bit till I was sure I passed 175 posts, and there were still waaaay more showing up. So I'm not sure why tumblthree stopped at such a short number. I'm not sure what extra info I can give. IT never errors out, so there isn't an error log to give info from. and other blogs will download completely, but tumblthreee seems to pick and choose which ones it'll completely download and others that it'll only download a handful.

**Comment by vicki-leekx** [comment]
Have you tried to do a clean install of TumblThree? 
Clean and delete all the files in C:\Users\pc\AppData\Local\TumblThree

Run disk cleanup in windows 10 and delete all your temporary files 
https://www.youtube.com/watch?v=mjedBDFtfU0

You can use CCleaner too 

Then after you've done that
download the latest version of TumblThree and try again 
authenticate to a Tumblr account before downloading 
Do the privacy content thing too. 
 If you still get errors try using another Tumblr account

Maybe you're downloading too fast and you're downloading too many blogs / files and the API is trying to slow you down
